modules/openapi-generator/src/main/resources/kotlin-client/build.gradle.mustache

```diff
@@ -57,6 +57,7 @@ dependencies {
     {{/moshi}}
     {{#gson}}
     compile "com.google.code.gson:gson:2.8.6"
+    compile "io.gsonfire:gson-fire:1.8.0"
```

Comment:
gson has own polymorhic implementation https://github.com/google/gson/blob/master/extras/src/main/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactory.java

Reply:
Yes, I'm aware of this.
Two reasons I didn't use it :
- gson-extra is not available in maven central
- I just backport the working logic from the java side